package.json

```diff
@@ -4,6 +4,7 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
+    "start": "react-scripts start",
```

Comment:
It looks like the start script ("react-scripts start") is included, but this project is configured to use Vite. Since Vite is already handling the development server via the dev script ("vite"), the start script referencing react-scripts is unnecessary.

Is there a specific reason for adding this?